packages/api/src/Database/Database.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { Connection, Repository } from 'typeorm';
+
+@Injectable()
+export class DatabaseService {
+  /**
+   * Initializes the database service
+   * @param connection The connection, which gets injected
+   */
+  constructor(@Inject('Connection') public connection: Connection) { }
+
+  /**
+   * Returns the repository of the given entity
+   * @param entity The database entity to get the repository from
+   */
+  async getRepository<T>(entity: string): Promise<Repository<T>> {
+    return this.connection.getRepository(entity);
+  }
+
+  /**
+   * Loops over all tables and removes everything!
+   * BE CAREFUL
+   * Used in testing
+   */
+  async DANGEROUSLY_RESET_DATABASE() {
+    Promise.all(this.connection.entityMetadatas.map(async entity => {
+      const repository = await this.getRepository(entity.name);
+      await repository.clear();
+    }));
+  }
```

Comment:
As I mentioned yesterday, I know nothing about testing, trying to learn now, but I have one question: Why would we want to reset the db in a test?

Reply:
@suzynakayama, I mentioned this on our talk last night. Was this still unclear? The intention behind resetting this is to make sure it's a "clean" set of data each time you test, so it doesn't introduce strange bugs each time. This should also be improved with #53

Reply:
@tristanMatthias , no I wrote this before the talk last night. thank you.

Reply:
OK great. Can you approve the PR then? I'll merge it in